src/BUILD

```diff
@@ -355,6 +363,20 @@ cc_library(
     alwayslink = 1,
 )
 
+#OPENVINO CALCULATORS
+cc_binary(
```

Comment:
Maybe add "model" into target name to cleary distinguish that those are just model files